src/app/components/NavBar/index.tsx

```diff
@@ -134,6 +134,7 @@ const NavigationBar = ({user, loggedOut, currentPath}: NavigationBarProps) =>
       </a>
       {loggedOut && <LoggedOutState currentPath={currentPath} />}
       {user && <LoggedInState user={user} currentPath={currentPath} />}
+      <Styled.PageTitleConfirmation />
```

Comment:
does it matter where in the document you put this? it doesn't read it in order or anything right? maybe put it up the tree a bit in Layout or Content or something? NavBar seems a bit weird

Reply:
Moved to Layout